--- xpyb-1.2/src/Makefile.am.orig	2009-12-17 19:46:14.000000000 +0100
+++ xpyb-1.2/src/Makefile.am	2010-07-02 23:30:35.605386711 +0200
@@ -11,8 +11,8 @@
 
 xcb_la_LIBADD = $(LIBXCB_LIBS)
 xcb_la_CPPFLAGS = -I$(PYTHON_INCLUDE)
-xcb_la_CFLAGS = -g $(CWARNFLAGS)
-xcb_la_LDFLAGS = -module
+xcb_la_CFLAGS = $(CWARNFLAGS)
+xcb_la_LDFLAGS = -module -avoid-version
 xcb_la_SOURCES = conn.c constant.c cookie.c error.c event.c except.c \
 		 ext.c extkey.c iter.c list.c module.c protobj.c \
 		 reply.c request.c response.c struct.c union.c void.c \
